src/pid/PIDConstants.java

```diff
@@ -0,0 +1,5 @@
+package pid;
+
+public class PIDConstants {
+  final static int TIMEOUT = 20;
```

Comment:
Call this something more specific to CTRE, as this can get confused easily with the PID running interval. Maybe `CTRE_DEVICE_CALLS_TIMEOUT`?